lib/ansible/modules/cloud/digital_ocean/digital_ocean_sshkey_facts.py

```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+#
+# Copyright: Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {'status': ['preview'],
+                    'supported_by': 'community',
+                    'metadata_version': '1.1'}
+
+
+DOCUMENTATION = '''
+---
+module: digital_ocean_sshkey_facts
+short_description: DigitalOcean SSH keys facts
+description:
+     - Fetch DigitalOcean SSH keys facts.
+version_added: "2.5"
+author: "Patrick Marques (@pmarques)"
+options:
+  oauth_token:
+    description:
+     - DigitalOcean API token.
+    required: true
+
+notes:
+  - Version 2 of DigitalOcean API is used.
+requirements:
+  - "python >= 2.6"
+'''
+
+
+EXAMPLES = '''
+- digital_ocean_sshkey_facts:
+    oauth_token: "{{ my_do_key }}"
+
+- set_fact:
+    pubkey: "{{ item.public_key }}"
+  with_items: "{{ ssh_keys|json_query(ssh_pubkey) }}"
+  vars:
+    ssh_pubkey: "[?name=='ansible_crtl']"
+
+- debug:
+    msg: "{{ pubkey }}"
+'''
+
+
+RETURN = '''
+# Digital Ocean API info https://developers.digitalocean.com/documentation/v2/#list-all-keys
```

Comment:
FYI comments will not be rendered in the RETURN table, if this is for future developers of this module, then that's fine.

If however this link is for users of the module then you may wish to consider adding it to `DOCUMENTATION:` `notes:`

http://docs.ansible.com/ansible/dev_guide/developing_modules_documenting.html

Reply:
This is a comment/reference where to check the API examples.